### PR TITLE
feat: auto reload browser on ADD change

### PR DESCRIPTION
**Pull request by tu1ly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ Options:
   [--server], [--no-server]  # Start standalone web server on port 8080
   [--port=PORT]              # Set port for --server option
   [--host=HOST]              # Set host for --server option
+  [--watch], [--no-watch]    # Reload API documentation when API Description Document changed
```
Comment (JackuB) [inline README.md]: How do I know what's the default? There should be only 1 flag IMHO

File: lib/apiary/command/preview.rb [hunk 2]
```diff
@@ -29,13 +30,17 @@ def initialize(opts)
       @options.proxy        ||= ENV['http_proxy']
       @options.server       ||= false
       @options.json         ||= false
+      @options.watch        ||= false
+      @options.interval     ||= 2000
```
Comment (JackuB) [inline lib/apiary/command/preview.rb]: Isn't it too much?

File: lib/apiary/file_templates/preview.erb [hunk 3]
```diff
@@ -10,6 +10,38 @@
...
+                            if (xhrData.readyState == 4 && xhrData.status == 200) {
+                                embed.iframeElement.contentWindow.postMessage({"origin": embed.ORIGIN, "eventType": "anonymousPreview", "data": {"code": xhrData.responseText}}, '*');
+                            }
+                        }
+                    }
+                }
+            }
+        }, <%= data[:interval] %>);
+
+
```
Comment (JackuB) [inline lib/apiary/file_templates/preview.erb]: Why so many newlines here?

File: lib/apiary/version.rb [hunk 4]
```diff
@@ -1,3 +1,3 @@
 module Apiary
-  VERSION = '0.6.1'.freeze
+  VERSION = '0.6.2'.freeze
```
Comment (JackuB) [inline lib/apiary/version.rb]: If this is following semver, it should bump to `0.7.0` because it's adding functionality

File: README.md (hunk 1 above)
Comment (tu1ly) [inline README.md]: it is generated from https://github.com/apiaryio/apiary-client/pull/142/files#diff-969dbf95872b9f711800ef6cb6b40992R28 `no-` option is added automatically for bool arg. default is false (== --no-watch)

File: lib/apiary/command/preview.rb (hunk 2 above)
Comment (tu1ly) [inline lib/apiary/command/preview.rb]: do you think 1 is better ? (i don't want add another arg option)

File: lib/apiary/version.rb (hunk 4 above)
Comment (tu1ly) [inline lib/apiary/version.rb]: changed

File: lib/apiary/file_templates/preview.erb (hunk 3 above)
Comment (tu1ly) [inline lib/apiary/file_templates/preview.erb]: fixed

**Comment by tu1ly** [comment]
@JackuB all comments addressed

File: lib/apiary/command/preview.rb (hunk 2 above)
Comment (JackuB) [inline lib/apiary/command/preview.rb]: I think 1s should be fine

File: README.md (hunk 1 above)
Comment (JackuB) [inline README.md]: Then we should let the user know what's the default behaviour

Comment (tu1ly) [inline README.md]: imho it is not needed - it is std for https://github.com/erikhuda/thor/wiki and the same as for other args

Comment (JackuB) [inline README.md]: I think there should be `changes` instead of `changed`

Comment (tu1ly) [inline README.md]: or 'has changed' maybe

**Pull request merged**